bedrock/firefox/views.py

```diff
@@ -553,6 +553,8 @@ def get_template_names(self):
                 template = 'firefox/whatsnew/index.html'
         elif locale == 'id':
             template = 'firefox/whatsnew/index-lite.id.html'
+        elif version.startswith('72.') and lang_file_is_active('firefox/whatsnew_72', locale):
+            template = 'firefox/whatsnew/whatsnew-fx72.html'
```

Comment:
Adding a [test](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/tests/test_base.py#L159) to make sure that the right template is shown here would be good too?